tip-wlan/charts/opensync-mqtt-broker/templates/statefulset.yaml

```diff
@@ -76,13 +76,35 @@ spec:
           readinessProbe:
             tcpSocket:
               port: {{ .Values.service.port1 }}
-            initialDelaySeconds: {{ .Values.probes.readinessProbe.initialDelaySeconds }}
-            timeoutSeconds: {{ .Values.probes.readinessProbe.timeoutSeconds }}
+            initialDelaySeconds: {{ .Values.probes.livenessProbe.initialDelaySeconds }}  
```

Comment:
Did we change the `.Values.probes.readinessProbe.` to `.Values.probes.livenessProbe.` under readinessProbe Or is it just my diff-viewer not showing correctly?